tests/queries/0_stateless/03403_function_tokenizeChinese.sql

```diff
@@ -0,0 +1,47 @@
+SELECT 'With constant values.';
+SELECT tokenizeChinese('他来到了网易杭研大厦');
+SELECT length(tokenizeChinese('他来到了网易杭研大厦'));
```

Comment:
@hanfei1991 Are the tokenized results okay or garbage?

Reply:
LGTM, it just copy/paste some tests from "jieba" repo :)